R/net.mod.simnet.R

```diff
@@ -48,23 +48,42 @@ sim_nets_t1 <- function(x, dat, nsteps) {
     }
 
     set.control.stergm <- get_control(dat, "set.control.stergm")
-
-    # Simulate dynamic network
-    suppressWarnings({
-      sim <- simulate(nw,
-                      formation = x$formation,
-                      dissolution = x$coef.diss$dissolution,
-                      coef.form = x$coef.form,
-                      coef.diss = x$coef.diss$coef.crude,
-                      time.slices = nsteps,
-                      time.start = 1,
-                      time.offset = 0,
-                      constraints = x$constraints,
-                      monitor = nwstats.formula,
-                      nsim = 1,
-                      control = set.control.stergm)
-      sim <- networkDynamic::activate.vertices(sim, onset = 1, terminus = Inf)
-    })
+    set.control.tergm <- get_control(dat, "set.control.tergm")
+
+    if (!is.null(set.control.stergm)) {
+      # Simulate dynamic network
+      suppressWarnings({
```

Comment:
Why are we suppressing the warnings here? 
(pure curiosity as we were doing it prior to this PR as well)

Reply:
I think there are time stamp warnings in some of the `simulate` calls in `simnet`.  It may be possible to avoid them in typical use cases by adjusting `time.start` and `time.offset`.

Reply:
thanks for the explanation. I don't think it's a necessity to address that here though.